Geometry/DTGeometryBuilder/test/python/validateDTGeometry_cfg.py

```diff
@@ -37,7 +37,7 @@
 
 process.valid = cms.EDAnalyzer("DTGeometryValidate",
 
-                               infileName = cms.untracked.string('cmsRecoGeom-2021.root'),
+#                               infileName = cms.untracked.string('cmsRecoGeom-2021.root'),
```

Comment:
see my comment https://github.com/cms-sw/cmssw/pull/31043/files/2d3b727f5f67fa52bda586c67dc5acbde80f3e77..278777a07be31745322a4ac08abba9488fac5af1#r478965333
its better to keep it configurable. So I would suggest that keep `infileName` and set it to
```
infileName = cms.untracked.string('Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root'),
```

Reply:
@smuzaffar : ok, but in validateDTGeometry_cfg.py do I have to write the complete path? infileName = cms.untracked.string('Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root') instead infileName = cms.untracked.string('cmsRecoGeom-2021.root') ?

Reply:
> So I would suggest that keep `infileName` and set it to
> 
> ```
> infileName = cms.untracked.string('Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root'),
> ```

Note that `FileInPath` can be used in python as well, so this could be
```python
infileName = cms.untracked.FileInPath('Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root'),
```
and in the C++
```c++
auto fp = iConfig.getUntrackedParameter<edm::FileInPath>("infileName");
```